### PR TITLE
State::PlayerPanel replaced with UI::PlayerPanel

### PR DESCRIPTION
**Pull request by alexeevdv**

### CODE REVIEW

File: src/UI/PlayerPanel.cpp [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+            {
+                openPipBoy();
+            }
+            break;
+        case SDLK_z:
+            openPipBoy(); // @TODO: go to clock
+            break;
+        case SDLK_ESCAPE:
+        case SDLK_o:
+            openGameMenu();
```
Comment (phobos2077) [inline src/UI/PlayerPanel.cpp]: I think some key presses that are not directly tied to PlayerPanel can be moved to State::Location. Like GameMenu, SaveGame, LoadGame, Exit,

Comment (alexeevdv) [inline src/UI/PlayerPanel.cpp]: Yep, but that is the whole new story....

File: src/UI/PlayerPanel.h [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    Size size() const override;
+    void render(bool eggTransparency = false) override;
+    void handle(Event::Event *event) override;
+    void think() override;
+
+    void playWindowOpenSfx();
+
+private:
+
+    std::shared_ptr<Image> _background;
```
Comment (phobos2077) [inline src/UI/PlayerPanel.h]: Why do you need shared_ptr?

File: src/UI/PlayerPanel.cpp [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+{
+
+PlayerPanel::PlayerPanel() : UI::Base()
+{
+    auto game = Game::getInstance();
+    auto renderer = game->renderer();
+    auto mouse = game->mouse();
+
+    _background = std::make_shared<Image>("art/intrface/iface.frm");
+    _ui.push_back(_background);
```
Comment (phobos2077) [inline src/UI/PlayerPanel.cpp]: Seems like it shares ownership with itself :)

**Pull request merged**